Bottom: 6aee362ac3b39d92a763c6098190db8ef935acc6
Top:    9b1055d0110ec6849a41fce135a424231d6d0c15
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-31 00:27:43 -0400

ADC register defines


---

diff --git a/arch/arm/mach-lpc31xx/include/mach/registers.h b/arch/arm/mach-lpc31xx/include/mach/registers.h
index 45af7fc..01f4114 100644
--- a/arch/arm/mach-lpc31xx/include/mach/registers.h
+++ b/arch/arm/mach-lpc31xx/include/mach/registers.h
@@ -276,6 +276,14 @@
  * ADC_REG register definitions
  **********************************************************************/
 #define ADC_CON_REG            __REG (ADC_PHYS + 0x20)
+#define ADC_R0_REG             __REG (ADC_PHYS + 0x00)
+#define ADC_R1_REG             __REG (ADC_PHYS + 0x04)
+#define ADC_R2_REG             __REG (ADC_PHYS + 0x08)
+#define ADC_R3_REG             __REG (ADC_PHYS + 0x0C)
+#define ADC_CSEL_REG           __REG (ADC_PHYS + 0x24)
+#define ADC_INT_ENABLE_REG     __REG (ADC_PHYS + 0x28)
+#define ADC_INT_STATUS_REG     __REG (ADC_PHYS + 0x2C)
+#define ADC_INT_CLEAR_REG      __REG (ADC_PHYS + 0x30)
 
 /***********************************************************************
  * SYS_REG register definitions
